services/main.py

```diff
@@ -0,0 +1,12 @@
+from scrapper.send import send
+from crawler.main import get_prod_url
+
+base_url = 'https://www.x-kom.pl'
+category_url = '/g-5/c/345-karty-graficzne.html?page=1'
```

Comment:
Dodaj to do pliku konfiguracyjnego i z tamtąd pobieraj wartości na branchu api znajdziesz przykład takiego czegoś